### ISSUE TITLE
Video files from failed import

### ISSUE THREAD

**Issue by roere** [Chocobozzz/PeerTube#3996]
I have a video file in my storage/videos folder from a failed import. Obviously the file was already imported, somehow the process later produced an error, transcoding was not finished and the originally imported file was left in the folder, seemingly with no entry in the database, as it is not accessible on the website or via the api, using the video uid. 
Do I have to delete these files manually or is there a "cleaner" way?

**Comment by Chocobozzz** [comment]
Hi,

What was the error?

**Comment by roere** [comment]
Still trying to reproduce it, that's why I just asked for the recommended way for deleting that unused video file.
Was one of this two problems I had:
1. The video import dialog (from url) didn't continue to the meta data page, just stayed on the page where you enter the url (after some time loading). I clicked the import button again and was brought to the meta data page then. Later I found in the logs, that the import failed, because the object/file already existed. My guess was, that, also it did not continue to the meta data page on the first try, the import nevertheless had started.
2. Again I used video import from url, finished the process, also adding meta data, without problems. Got a message, that the video was imported successfully (also Log status "completed" for that job). Later the transcoding failed (same error I mentioned here:  #3926) and I also got the message that import for that video failed.

I give u an update on that.

Do I delete the file manually?
